omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -371,12 +371,13 @@
 default['private_chef']['nginx']['ssl_certificate'] = nil
 default['private_chef']['nginx']['ssl_certificate_key'] = nil
 default['private_chef']['nginx']['ssl_dhparam'] = nil
-default['private_chef']['nginx']['ssl_country_name'] = "US"
-default['private_chef']['nginx']['ssl_state_name'] = "WA"
```

Comment:
The cert only requires 4 fields:
common_name (CN)
Org (O)
Org_unit (OU)
Country (C)

The additional fields can be nil without problem. Also the LWRP doesn't support them, so they became unreferenced by the changeover.

Also also since anything this recipe generates is going to be a self-signed cert, having the additional fields won't make things any better.